substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfInfoSectionImpl.java

```diff
@@ -326,10 +1221,93 @@ public int writeAttrString(String value, byte[] buffer, int p) {
         }
     }
 
+    public int writeAttrAccessibility(int modifiers, byte[] buffer, int p) {
+        byte access;
+        if (Modifier.isPublic(modifiers)) {
+            access = DW_ACCESS_public;
+        } else if (Modifier.isProtected(modifiers)) {
+            access = DW_ACCESS_protected;
+        } else if (Modifier.isPrivate(modifiers)) {
+            access = DW_ACCESS_private;
+        } else {
+            // package private -- make it public for now
+            access = DW_ACCESS_public;
+        }
+        return writeAttrData1(access, buffer, p);
+    }
+
+    public int writeOopRelocationExpression(byte[] buffer, int p) {
+        int pos = p;
+        if (dwarfSections.useHeapBase()) {
+            // oop is 32 bit signed offset, possibly shifted by CompressEncoding.getShift()
```

Comment:
Use the
```
            /*-
             *
             *
             */
```
method then `eclipseformat` will not reformat your comment content. (Note the `-` after `/*`)

Reply:
Yes, thanks for the tip.

Done.